src/coreclr/tools/aot/Mono.Linker.Tests.Cases/DynamicDependencies/Dependencies/AssemblyDependency.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Mono.Linker.Tests.Cases.DynamicDependencies.Dependencies
```

Comment:
Do we want to duplicate all these tests? Wouldn't be better to have that in some shared location?

Reply:
I think if you put the files in the right location it'd be enough and save us from deleting them all in few weeks/months

Reply:
I think the tricky bit is having them in the right location (don't think we agreed on one yet), and I don't want to have that discussion as part of a random unrelated PR.